app/routes/public/sessions/list.js

```diff
@@ -16,169 +16,64 @@ export default Route.extend({
   },
   async model(params) {
     const eventDetails = this.modelFor('public');
-    let sessions =  null;
-    if (params.session_status === 'today') {
-      sessions = await this.store.query('session', {
-        filter: [
+    const period = params.session_status === 'today' ? 'day' : params.session_status;
+    let filterOptions = [
```

Comment:
```suggestion
    const filterOptions = [
```

Reply:
Filter options are changing in the if conditional, const will not be useful then.

Reply:
Try it

Reply:
ESLint passed 😮